src/app/templates/SeedPhraseInput/index.tsx

```diff
@@ -39,10 +38,11 @@ export const SeedPhraseInput: FC<SeedPhraseInputProps> = ({
   const { popup } = useAppEnv();
 
   const [pasteFailed, setPasteFailed] = useState(false);
-  const [draftSeed, setDraftSeed] = useState(new Array(defaultNumberOfWords).fill(''));
-  const [showSeed, setShowSeed] = useState(true);
+  const [draftSeed, setDraftSeed] = useState(new Array<string>(defaultNumberOfWords).fill(''));
```

Comment:
It is considered bad practice to use the Array constructor because it works differently depending on the number of arguments, and you need to remember this

Reply:
Maybe in this case it is ok but I'm not sure